lib/ts/recipe/session/recipe.ts

```diff
@@ -74,7 +74,7 @@ export default class SessionRecipe extends RecipeModule {
 
         this.isInServerlessEnv = isInServerlessEnv;
 
-        if (this.config.jwt.enable === true) {
+        if (this.config.jwt.enable) {
```

Comment:
this is an unrelated change. Please revert it :)

Reply:
Ok, will do